### PR TITLE
Clean up TagPrefix::getMaterialAmount

### PR DESCRIPTION
**Pull request by Jorgsaa**
Wanted to start contributing here, so i read and found this method (`TagPrefix::getMaterialAmount`) that was a bit hard to follow.
Also, what do you guys think about statically importing `GTValues.M` here?

Any suggestions are appreciated :)

### CODE REVIEW

**Comment by Jorgsaa** [comment]
- [ ] Change  `Map<Material, Long>` to `Map<UnificationEntry, Long>` due to possible conflicts between tagprefixes in the future (@Yefancy)

**Review by Yefancy — APPROVED** [review]
